### ISSUE TITLE
Error in schema for cp_mgmt_access_rules_module for VPN

### ISSUE THREAD

**Issue by Smixi** [CheckPointSW/CheckPointAnsibleMgmtCollection#107]
Hi,

I'm trying to add a rule using this Ansible module.

I figured out that rules that I push that are not using the VPN attribute are created, but those using it fail.

E.g :
```yaml
- name: Create access rules
  check_point.mgmt.cp_mgmt_access_rules:
    layer: "{{item.layer}}"
    rules: 
      - name: 'RuleName'
        source: 
          - 'RuleSource'
        destination: 
          - 'RuleDest'
        vpn: 
          - community: 
              - name: Any
```
The documentation is unclear of what the community object must look like. I expected something like above, but it returned 

``` 
msg: "value of vpn must be one of: Any, All_GwToGw, got: [{'community': [{'name': 'Any'}]}]"}}
```

So I tried to use something like 

`        vpn: 
          - Any
`

or even 
`        vpn: Any
`
which outputs : "msg": "dictionary requested, could not parse JSON or key=value"} 

It's seems that the webservice documentation uses the later, which is not parsed correctly by ansible because it is defined as having dict elements. 

Do you have any recommandation ?

Thanks in advance :).

**Comment by Smixi** [comment]
Hi @chkp-edenbr,

I  can't find any reference about "Any" or "All_GwToGw" in the module. As described, using 
`vpn: Any
` fails during parsing by ansible:  
`"msg": "dictionary requested, could not parse JSON or key=value"`, which I guess is because the type is list, and not string, by [this line](https://github.com/CheckPointSW/CheckPointAnsibleMgmtCollection/blob/ae098ac21186acd76846c44448740208787cc9f7/plugins/modules/cp_mgmt_access_rules.py#LL216C8-L216C8)

As I mentionned too, using the documented suboptions are parsed by the module, but the API refuse to works with and reject the request.

Do you have any working example I can base upon, or tell me where in the tasks the provided parameters are wrong ?

Regards,

Sami.

**Comment by chkp-edenbr** [comment]
Hi @Smixi 
Fix has been added and will be available in the upcoming release.
Thanks,
Eden

**Comment by chkp-edenbr** [comment]
Issue fixed in new release v5.1.0.
Thanks,
Eden
